testing/tests/DevExpress.ui.widgets.form/form.markup.tests.js

```diff
@@ -226,11 +227,27 @@ QUnit.module('Form', () => {
             ]
         });
 
-        assert.notOk($formContainer.find('.' + FIELD_ITEM_CLASS + ' .dx-texteditor').hasClass('dx-state-readonly'), 'editor isn\'t read only');
+        assert.notOk($formContainer.find('.' + FIELD_ITEM_CLASS + ' .dx-texteditor').hasClass(READONLY_STATE_CLASS), 'editor isn\'t read only');
 
         $formContainer.dxForm('instance').option('readOnly', true);
 
-        assert.ok($formContainer.find('.' + FIELD_ITEM_CLASS + ' .dx-texteditor').hasClass('dx-state-readonly'), 'editor is read only');
+        assert.ok($formContainer.find('.' + FIELD_ITEM_CLASS + ' .dx-texteditor').hasClass(READONLY_STATE_CLASS), 'editor is read only');
+    });
+
+    test('editor should not change readonly state after update readOnly option in editor and then in form', function(assert) {
```

Comment:
It should be changed after editorOptions.readOnly change, but not after form readOnly change
```suggestion
    test('editor should not change readonly state after form readOnly option change if editorOptions.readOnly was updated before', function(assert) {
```